modules/UI/ring_overlay/RingOverlay.js

```diff
@@ -60,6 +70,8 @@ class RingOverlay {
     _getHtmlStr(callee) {
         let callingLabel = this.isRinging? "<p>Calling...</p>" : "";
         let callerStateLabel =  this.isRinging? "" : " isn't available";
+        let audioHTML = this.dontPlayAudio? "" :
```

Comment:
I think that the new formatting rules would request a space before the ? and the : should be on the new line together with what follows them.